Week3/homework/step2-4.js

```diff
@@ -2,6 +2,7 @@
 
 function Dog() {
   // add your code here
+  (this.name = "Da Kin'"), (this.color = 'blue'), (this.numLegs = 3);
```

Comment:
Good, you could also write it like this;
```
this.name = "Da Kin'";
this.color = "blue";
this.numLegs = 3;
```

Looks a bit more readable in my opinion.